perllib/FixMyStreet/Cobrand/HighwaysEngland.pm

```diff
@@ -198,32 +220,24 @@ sub report_new_is_on_he_road {
         srsname => "urn:ogc:def:crs:EPSG::4326",
         typename => "Highways",
         filter => "<Filter><DWithin><PropertyName>geom</PropertyName><gml:Point><gml:coordinates>$x,$y</gml:coordinates></gml:Point><Distance units='m'>15</Distance></DWithin></Filter>",
+        accept_feature => sub { 1 },
     };
+
     my $ukc = FixMyStreet::Cobrand::UKCouncils->new;
     my $features = $ukc->_fetch_features($cfg, $x, $y);
-    $cfg->{accept_feature} = sub {
-        my $feature = shift;
-        return $feature->{properties}->{feature_ty} = 'ROAD'
-        };
     my $nearest = $ukc->_nearest_feature($cfg, $x, $y, $features);
     # National Highways responsible for litter on Motorways and AM roads
+    # And doesn't matter if we are not on a NH road
     if ($nearest && $nearest->{properties}->{'ROA_NUMBER'} =~ /^(M|A\d+M)/) {
         $self->{c}->stash->{report_new_is_on_he_road_not_litter} = 0;
-    } elsif (scalar @$features) {
-        $self->{c}->stash->{report_new_is_on_he_road_not_litter} = $self->_report_new_is_on_he_road_not_litter;
```

Comment:
This was being run on every /around page; I kept most of this the same but moved the setting of this to in national_highways_cleaning_groups so it would only run on /report/new when needed.